docker/mcp/app.py

```
@@ -0,0 +1,94 @@
+@app.get("/health")
+def health_check():
+    """Health check endpoint."""
+    return {"status": "healthy", "service": "mcp"}
+ 
+@app.post("/api/search", response_model=SearchResponse)
+def search(request: QueryRequest):
+    """Search the web and return results with content."""
+    try:
+        global SERPER_API_KEY
+        query = request.query
+        
```

Pylint (3.3.7)

[convention] Trailing whitespace

(C0303)